README.md

```diff
@@ -232,7 +149,7 @@ the output to a log file or other log management system.  See the
 [signalfx-agent command](./docs/signalfx-agent.1.man) doc for more information on
 supported command flags.
 
```

Comment:
Should they now continue with the Confirmation step? If so, add something like To confirm you are receiving data, see _Confirmation_ (link). 

Same applies to the Windows zip instructions.

Reply:
Moved the tar and zip back into the README. Put Confirmation as a step in the Quick Install.